src/components/StickerCreator/stickerBackgroundColors.ts

```diff
@@ -0,0 +1,7 @@
+import type { ProductConfig } from '../../types/product.ts';
+
+export const stickerBackgroundColor: Record<ProductConfig['color'], string> = {
+  'color-cheap': '#FBBD08',
+  'color-average': '#189cab',
+  'color-expensive': '#ed5c01',
+};
```

Comment:
Let's use our theme's colors so it does not look like random colors:


```suggestion
import type { ProductConfig } from '../../types/product.ts';
import { theme } from '../../theme.ts';

export const stickerBackgroundColor: Record<ProductConfig['color'], string> = {
  'color-cheap': theme.palette.warning.main,
  'color-average': theme.palette.info.main,
  'color-expensive': theme.palette.primary.main,
};
```

Reply:
50/50 this will work without types errors

Reply:
Ok in my previous attempt it does not work, now it works!